.github/workflows/build.yml

```diff
@@ -42,13 +42,19 @@ jobs:
       # don't compile with all combinations of features.
       - name: Clippy
         run: cargo clippy --workspace --all-features --all-targets -- -D warnings
-      
-      - name: Test
+
+      - name: Test with embedded-db
```

Comment:
Would be nice to make these separate jobs rather than separate steps so they could run in parallel